exec-agent/docs/ws_api.md

```diff
@@ -0,0 +1,329 @@
+Websocket API
+---
+
+### Process API
+
+#### Start process
+
+##### Request
+
+- __name__ - the name of the command
+- __commandLine__ - command line to execute
+- __type__(optional) - command type
```

Comment:
Who knows, we don't use it right now but we may use it in future, so i would probably let it as it is